### PR TITLE
[cb-fetch_v1.10.x] Add cb-fetch libdef

### PR DESCRIPTION
**Pull request by Mouvedia**
- Links to documentation: https://github.com/Mouvedia/cb-fetch/blob/master/README.md
- Link to GitHub or NPM: https://github.com/Mouvedia/cb-fetch
- Type of contribution: new definition

Other notes:

I am gonna release 1.10.0 on npm next month and I want the libdef to be ready by then.

Related:

facebook/flow#7386

### CODE REVIEW

File: definitions/npm/cb-fetch_v1.10.x/test_cb-fetch_v1.10.x.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+request().hook('upload', () => {});
+
+// $ExpectError
+request().get().send();
+
+request().done({
+    error(response) {
+        // $ExpectError
+        response.headers.hasOwnProperty();
+    }
```
Comment (pascalduez) [inline definitions/npm/cb-fetch_v1.10.x/test_cb-fetch_v1.10.x.js]: Could you please wrap tests in a few [`describe/it` blocks](https://github.com/flow-typed/flow-typed/blob/master/CONTRIBUTING.md#use-describe-and-it-blocks-to-limit-scope)?

**Comment by pascalduez** [comment]
Side question for curiostiy: since your the library maintainer, why not shipping the Flow definitions along with the package?

**Comment by Mouvedia** [comment]
> …why not shipping the Flow definitions along with the package?

1. I didn't find any IDE/plugin combination that automatically picked `js.flow` files, despite my effort
2. the documentation is missing: #3609

**Comment by Mouvedia** [comment]
@pascalduez I have wrapped the tests.

> …why not shipping the Flow definitions along with the package?

1. I didn't find any IDE/plugin combination that automatically picked `js.flow` files, despite my effort
2. the documentation is missing: facebook/flow#3609

**Comment by pascalduez** [comment]
> I didn't find any IDE/plugin combination that automatically picked .js.flow files, despite my effort

That's Flow's job, not IDE/plugins I would say. At least I can attest is works in Atom, Vim, Code.
Anyway, thanks for the PR!

**Comment by Mouvedia** [comment]
> At least I can attest is works in Atom, Vim, Code.

I tried WebStorm, VS code and Atom; none were able to pick it up*. I wish you could document which IDE/plugin worked for you.

<sub>\* `import for from 'bar'` should automatically pick the types if the `<main>.flow` file corresponds to the `main` field</sub>

File: definitions/npm/cb-fetch_v1.10.x/test_cb-fetch_v1.10.x.js (hunk 1 above)
Comment (Mouvedia) [reply definitions/npm/cb-fetch_v1.10.x/test_cb-fetch_v1.10.x.js]: How do you display the output using flow-typed? Why bother if it's not used?
I couldn't find a `--verbose` option.

**Pull request merged**